teamB/cheonsongin/services/userService.js

```diff
@@ -0,0 +1,24 @@
+const userDao = require('../models/userDao');
+
+const signUp = async (name, email, password, profileImage) => {
+  const pwValidation = new RegExp(
+    '^(?=.*[A-Za-z])(?=.*[0-9])(?=.*[!@#$%^&*])(?=.{8,20})'
+  );
```

Comment:
자바스크립트에서 문자열을 변수에 담을 때 변수는 대문자로 작성하는 컨벤션이 있습니다. 참고해주세요~!
```suggestion
  const PASSWORD_REGEX = new RegExp(
    '^(?=.*[A-Za-z])(?=.*[0-9])(?=.*[!@#$%^&*])(?=.{8,20})'
  );
```